CMakeLists.txt

```diff
@@ -477,19 +474,14 @@ if(UNIX AND NOT CMAKE_COMPILER_IS_GNUCXX)
 	set(CMAKE_EXE_LINKER_FLAGS "-lstdc++ -lm ${CMAKE_EXE_LINKER_FLAGS}")
 endif(UNIX AND NOT CMAKE_COMPILER_IS_GNUCXX)
 
-# compose datadir path of datarootdir and datadirname
-set(DATADIR ${DATAROOTDIR}/${DATADIRNAME})
-
 if(NOT WIN32)
-	add_definitions(-DWESNOTH_PATH="${DATADIR}")
+	add_definitions(-DWESNOTH_PATH=\\\"${CMAKE_INSTALL_FULL_DATADIR}/${DATADIRNAME}\\\")
```

Comment:
CMP005 has been eliminated. Remove the escapes (\\\) on both ends they are no longer required.

This is causing a Travis faliure.